electrumx/server/block_processor.py

```diff
@@ -402,34 +402,37 @@ def advance_txs(self, txs):
         spend_utxo = self.spend_utxo
         undo_info_append = undo_info.append
         update_touched = self.touched.update
-        hashXs_by_tx = []
-        append_hashXs = hashXs_by_tx.append
 
-        for tx, tx_hash in txs:
-            hashXs = []
-            append_hashX = hashXs.append
-            tx_numb = s_pack('<I', tx_num)
+        hashXs_by_tx = [set() for _ in txs]
```

Comment:
Why not `hashXs_by_tx = [set() for _ in range(len(txs))]`?

Reply:
I'm near certain you're right and that as-written is fastest